packages/solid/components/Label/Label.styles.ts

```diff
@@ -29,11 +29,11 @@ const container: LabelConfig = {
   },
   base: {
     display: 'flex',
-    justifyContent: 'center',
+    justifyContent: 'flexStart',
```

Comment:
why are we changing this one, just a stylistic choice?

Reply:
flexStart is the default. Center requires label to have a width.